overlay/backend/services/generate-delta-worker/tests/docker-compose.yml

```
@@ -0,0 +1,15 @@
+name: acceptance
+services:
+  generate-delta-worker:
+    build:
+      args:
+        - BIN_FILE=backend/tests/bin/generae-delta-worker.test
+    image: generate-delta-worker:test
+    entrypoint: "/bin/bash"
+    command: "/test_generate_delta/test_generate_delta.sh"
+    environment:
+      GOCOVERDIR: /cover
+    user: ${UID:-0}:${GID:-0}
+    volumes:
+      - ./cover:/cover
```

There is a shared coverage directory in `backend/services/tests/cover`. The directory must exist for the service to collect coverage.
```suggestion
      - ../../../tests/cover:/cover
```